tools/docs/schema.ts

```diff
@@ -175,6 +175,7 @@ export async function generateSchema(
     title: `JSON schema for Renovate ${version} config files (https://renovatebot.com/)`,
     $schema: 'http://json-schema.org/draft-04/schema#',
     'x-renovate-version': `${version}`,
+    allowComments: true,
```

Comment:
Is this a magic field in a JSON Schema that VS Code understands? Or is it standard JSON Schema?

Reply:
Maybe I should have advertised this PR as _avoid VS Code flagging comments_ rather than _avoid editors flagging comments_, if that's your point. 😅